docs/metasploit-framework.wiki/ad-certificates/Attacking-AD-CS-ESC-Vulnerabilities.md

```diff
@@ -335,9 +335,9 @@ With this we now have a list of certificates that can be utilized for privilege
 
 # Using the ESC1 Vulnerability To Get a Certificate as the Domain Administrator
 Getting a certificate as the current user is great, but what we really want to do is elevate privileges if we can.
-Luckily we can also do this with the `icpr_cert` module. We just need to also set the `ALT_UPN` option to specify who we
-would like to authenticate as instead. Note that this only works with ESC1 vulnerable certificate templates which is why
-we can do this here.
+Luckily we can also do this with the `icpr_cert` module. We just need to also set the `ALT_SID` and `ALT_UPN` options to
+specify who we would like to authenticate as instead. Note that this only works with certificate templates that are
+vulnerable to ESC1 due to having the `CT_FLAG_ENROLLEE_SUPPLIES_SUBJECT` flag set.
```

Comment:
Technically setting ALT_SID isn't *always* required. It's required today on fully patched servers with the registry key set to require the strong mapping and will be required by default in the future. To avoid providing an abundance of details, this simply mentions that it should be set. If the SID isn't required because the server isn't patched, then including it is harmless.